models/pattern/core/pattern.go

```diff
@@ -341,7 +341,9 @@ ManifestLoop:
 			}
 			return pattern, ErrParseK8sManifest(err)
 		}
-
+		if len(manifest) == 0 {
+			continue
```

Comment:
Is continuing the right thing to do? @Revolyssup @sudo-NithishKarthik

Reply:
Is there a response regarding an empty file being set and sent somewhere else?

Reply:
When an empty file is sent,this logic won't be executed as that yaml will contain 0 manifests and logic will never enter this loop

Reply:
Makes sense. Thanks for this.